client/src/components/Cropper/index.tsx

```diff
@@ -43,9 +41,12 @@ export default class Cropper extends React.Component<any> {
         this.props.createFile(preview);
     }
 
-    public onBeforeFileLoad = (elem: any): void => {
-        if (elem.target.files[0].size > this.FILE_SIZE) {
-            alert('File is too big!');
+    public onBeforeFileLoad = (
+        elem: any): void => {
+        this.props.checkFileSize(elem.target.files[0].size);
```

Comment:
move in expression elem.target.files[0].size > FILE_SIZE

Reply:
fixed